services/generic/client.go

```diff
@@ -0,0 +1,57 @@
+// Package generic contains a gRPC based generic client.
```

Comment:
The generic service code and tests are pretty much copy-pasted from the generic component code with `:%s/component*/service*/g`. Let me know if you think I shouldn't be copy-pasting anywhere...